controllers/admin/AdminImportController.php

```diff
@@ -1401,7 +1401,7 @@ protected function categoryImportOne($info, $default_language_id, $id_lang, $for
                 $this->errors[] = $this->trans(
                     'The category ID must be unique. It can\'t be the same as the one for the parent category (ID: %1$s).',
                     [
-                        !empty($info['id']) ? Tools::htmlentitiesUTF8($info['id']) : 'null',
+                        isset($info['id']) && !empty($info['id']) ? Tools::htmlentitiesUTF8($info['id']) : 'null',
```

Comment:
`isset` not needed here
```suggestion
                        !empty($info['id']) ? Tools::htmlentitiesUTF8($info['id']) : 'null',
```